internal/util-collection/src/main/scala/sbt/internal/util/TypeFunctions.scala

```diff
@@ -8,19 +8,22 @@
 package sbt.internal.util
 
 trait TypeFunctions {
+  import TypeFunctions._
   type Id[X] = X
   type NothingK[X] = Nothing
   sealed trait Const[A] { type Apply[B] = A }
   sealed trait ConstK[A] { type l[L[x]] = A }
   sealed trait Compose[A[_], B[_]] { type Apply[T] = A[B[T]] }
   sealed trait ∙[A[_], B[_]] { type l[T] = A[B[T]] }
+  type AnyLeft[T] = Left[T, Nothing]
```

Comment:
Type inference for the TypeFunctions.Ops extension class doesn't work without explicitly defining these types (which I know partially defeats the purpose of lambda syntax).